### PR TITLE
WIP - Add a basic ingress chart

### PR DESCRIPTION
**Pull request by pierreozoux**
This is to open the discussion after #727 

I created a new directory `common`, because I'm not sure that we should mix them with the `stable` one. They don't bring any functionalities, but are more building blocks.

### CODE REVIEW

**Comment by prydonius** [comment]
Thanks @pierreozoux! There's actually some work @technosophos has done over in https://github.com/technosophos/common-chart. I was thinking we could collaborate there and then eventually add a common chart into this repo.

File: common/ingress/values.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Configure the ingress object to hook into existing infastructure
+### ref : http://kubernetes.io/docs/user-guide/ingress/
+###
+ingress:
+  service:
+    name: example
+    port: 80
+  tls: false
+  hostname: example.com
```
Comment (archseer) [inline common/ingress/values.yaml:9]: I like the host handling better the way the concourse chart does it, you can specify a list of hosts: https://github.com/kubernetes/charts/blob/5ba28e43b253928e374529066c65ab12ff458c56/stable/concourse/values.yaml#L288-L292

Though I agree that in most cases I'd use a single host.

Comment (pierreozoux) [reply common/ingress/values.yaml:9]: Yes, actually, I'm working on GitLab, and this is something needed.

**Comment by viglesiasce** [comment]
@pierreozoux @prydonius can we close this out in favor of the work being done in common-chart repo which will eventually make it in here?

**Comment by viglesiasce** [comment]
Closing as stale. Please re-open if/when you have time to revisit.

**Pull request closed without merging**